### PR TITLE
IOS-9285 Fix radio button / check box colors

### PR DESCRIPTION
**Pull request by amegias**
## 🎟️ **Jira ticket**
https://jira.tid.es/browse/IOS-9285

## 🥅 **What's the goal?**
Fix the border color of these two components

## 🚧 **How do we do it?**
As @AnaMontes11 reported, we had to change the border color to `.control`

Captura de pantalla 2023-08-31 a las 17 43 44

Captura de pantalla 2023-08-31 a las 17 43 49

## 🏑 **AppCenter build**
https://install.appcenter.ms/orgs/tuenti-organization/apps/mistica-swiftui-ios/distribution_groups/public/releases/27

### CODE REVIEW

**Comment by amegias** [comment]
@AnaMontes11 I've also seen that CheckBox border is wrong so I've fixed. Please, confirm that I'm not wrong please :)
Thanks for reporting the bug!

File: Sources/Mistica/Components/RadioButton/RadioButton.swift [hunk 1]
```diff
@@ -151,7 +151,7 @@ private extension RadioButton {
             layer.borderWidth = intrinsicContentSize.width / 4.0
         } else {
             layer.backgroundColor = UIColor.background.cgColor
```
[the remark this answers is not in the document]
Comment (AnaMontes11) [reply Sources/Mistica/Components/RadioButton/RadioButton.swift:153]: @amegias You can omit this comment; we will evaluate it internally for potential consolidation.

**Review by AnaMontes11 — APPROVED** [review]
Nice! thanks for you quick work!

**Review by AnaMontes11 — APPROVED** [review]

**Review by WanaldinoTelefonica — APPROVED** [review]

**Pull request merged**